README.md

```diff
@@ -12,12 +12,13 @@ A production-ready Node.js connector for the Bybit APIs and WebSockets, with Typ
 
 ## Issues & Discussion
 - Issues? Check the [issues tab](https://github.com/tiagosiebler/bybit-api/issues).
+- RestClient not defined? With the recent addition of LinearClient, we renamed RestClient to InverseClient. Change all old references.
 - Discuss & collaborate with other node devs? Join our [Node.js Algo Traders](https://t.me/nodetraders) engineering community on telegram.
```

Comment:
Silly request:
```suggestion
- Discuss & collaborate with other node devs? Join our [Node.js Algo Traders](https://t.me/nodetraders) engineering community on telegram.
- `'bybit-api' has no exported member 'RestClient'`: use `InverseClient` instead of `RestClient`
```